Source/WebCore/platform/Length.cpp

```diff
@@ -284,7 +284,7 @@ static Length blendMixedTypes(const Length& from, const Length& to, const Blendi
 
 Length blend(const Length& from, const Length& to, const BlendingContext& context)
 {
-    if ((from.isAuto() || to.isAuto()) || (from.isUndefined() || to.isUndefined()))
+    if ((from.isAuto() || to.isAuto()) || (from.isUndefined() || to.isUndefined()) || (from.isNormal() || to.isNormal()))
```

Comment:
I suggest removing the parentheses that suggest a grouping that does not exist.

Reply:
Fixed.